src/service/iot_config.proto

```diff
@@ -348,6 +348,14 @@ message load_region_req_v1 {
 
 message load_region_res_v1 {}
 
+message gateway_location_req_v1 {
+  bytes gateway = 1;
+  uint64 timestamp = 2;
+  bytes signature = 3;
```

Comment:
Who is signing this and how would you know who’s signing?

Reply:
This is the same as the other messages only a handful of admin keys have access to this RPC

Reply:
perhaps add a comment here so it's not confusing next time someone comes across this?